src/ray/common/ray_config_def.h

```diff
@@ -326,11 +326,6 @@ RAY_CONFIG(int64_t, enable_metrics_collection, true)
 // Max number bytes of inlined objects in a task rpc request/response.
 RAY_CONFIG(int64_t, task_rpc_inlined_bytes_limit, 10 * 1024 * 1024)
 
-/// Maximum number of tasks that can be in flight between an owner and a worker for which
-/// the owner has been granted a lease. A value >1 is used when we want to enable
-/// pipelining task submission.
-RAY_CONFIG(uint32_t, max_tasks_in_flight_per_worker, 1)
```

Comment:
maybe leave it and mark it deprecated and get rid of it in next 1-2 releases

Reply:
I don't think this is necessary--- all Ray internal flags are considered private unless otherwise documented.

Reply:
Could you give me an example of public flag? I'm curious how we document it so that users know it's public

Reply:
It should be pretty obvious, e.g., have all caps version and a comment.